packages/experiments/src/components/Stack/Stack.tsx

```diff
@@ -5,22 +5,13 @@ import { IStackItemProps, IStackItemStyles } from './StackItem/StackItem.types';
 import { IStackProps, IStackStyles } from './Stack.types';
 import { styles } from './Stack.styles';
 import { mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+import { getNativeProps, htmlElementProperties } from 'office-ui-fabric-react/lib/Utilities';
```

Comment:
these imports probably should be relative paths from `../../Utilities` and Styling. This keeps the package dependency in one place.